app/javascript/components/util/DeleteDialog.js

```diff
@@ -78,6 +81,26 @@ class DeleteDialog extends React.Component {
               <div className="character-limit-text">{MAX_REASON_LENGTH - this.state.delete_reason_text.length} characters remaining</div>
             </React.Fragment>
           )}
+          {this.props.type === 'Lab Result' && this.props.promptSymptomOnset && (
+            <div>
+              <p>
```

Comment:
same comment as in the lab modal

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/files#diff-431caf4d60fa1325346adae1e9dd1b818a55ea030365a593cdf8ebcaaab58dbdR86